Userland/Libraries/LibWebView/WebContentClient.h

```diff
@@ -101,6 +101,7 @@ class WebContentClient final
     virtual void did_change_theme_color(u64 page_id, Gfx::Color color) override;
     virtual void did_insert_clipboard_entry(u64 page_id, String const& data, String const& presentation_style, String const& mime_type) override;
     virtual void did_change_audio_play_state(u64 page_id, Web::HTML::AudioPlayState) override;
+    virtual void did_update_navigation_buttons_state(u64 page_id, bool can_go_back, bool can_go_forward) override;
```

Comment:
nit: We call these `back_enabled` and `forward_enabled` everywhere else, let's not call them something differently only here